src/plugins/simple-pricing/resolvers/Product.js

```diff
@@ -0,0 +1,5 @@
+import xformProductPricing from "../xforms/xformProductPricing.js";
+
+export default {
+  pricing: (node) => xformProductPricing(node)
```

Comment:
Can we add a resolver for `price` that...
- Returns the `price` as is with no modification
- but, `Logger.warn` with a deprecation notice?

Reply:
updated